web-stories.php

```diff
@@ -47,6 +47,7 @@
 define( 'WEBSTORIES_PLUGIN_DIR_URL', plugin_dir_url( WEBSTORIES_PLUGIN_FILE ) );
 define( 'WEBSTORIES_ASSETS_URL', WEBSTORIES_PLUGIN_DIR_URL . 'assets' );
 define( 'WEBSTORIES_MINIMUM_PHP_VERSION', '5.6.0' );
+define( 'WEBSTORIES_MINIMUM_WP_VERSION', '5.3.0' );
```

Comment:
Instead of having these constants, we just use `get_plugin_data` and the `RequiresWP` and `RequiresPHP` values? I am a little worries these values will not remain in sync?

Reply:
`get_plugin_data` only supports `RequiresWP` and `RequiresPHP` since WP 5.3, so this won't work for older versions, for which this message is for

Reply:
We can backfill these headers using this [filter](https://github.com/WordPress/wordpress-develop/blob/f0733600c9b8a0833d7e63f60fae651d46f22320/src/wp-includes/functions.php#L6034-L6044). 

But that maybe overkill.